thirdparty/libbacktrace/amalgamation.txt

```diff
@@ -0,0 +1,4 @@
+The libbacktrace source is distributed here as an amalgamation (https://sqlite.org/amalgamation.html).
+This means that, rather than mirroring the entire libbacktrace repo here,
+a script was used to bundle all local includes together into a single C file,
```

Comment:
the script's source or a link to it may be added here

Reply:
hi, sorry for the late reply. @spaceface777 did it and i'm not sure if he has given a gist link for the script for this library.

Reply:
I actually didn't use a script to generate this amalgamation; i did it by hand using regex, mostly.